tasks/closedSourceRationaleCheck.ts

```diff
@@ -25,6 +25,12 @@ const getInvalidRepos = async (privateRepos: Repo[]): Promise<Repo[]> => {
     repo.readme = await getReadme(repo.name)
   })
   await Promise.all(promises)
+  
+  privateRepos.forEach(repo => {
+    if (repo.readme === "") {
+      console.error(`Repo has empty readme! ${repo.name}`)
```

Comment:
CI is probably failing because Jest thinks that `console.error` means the test should fail. I would mock it before the tests, like:

```tsx
beforeEach(() => {
  console.error = jest.fn()
})
```

Or something similar, I think?